wowup-electron/main.ts

```diff
@@ -274,7 +283,7 @@ function createWindow(): BrowserWindow {
       e.preventDefault();
       win.hide();
 
-      if (preferenceStore.get("collapse_to_tray") === true) {
+      if (preferenceStore.get(collapseToTrayKey) === true) {
```

Comment:
interestingly this would never work since its a string should be true.toString().

Reply:
It's actually using booleans here (verified that while debugging), because it's not the wrapper that does `toString()`:
```typescript
const preferenceStore = new Store({ name: "preferences" });
```

I could probably change this to use the `PreferenceStorageService` though?